### PR TITLE
Update opm in dev environment to v1.16.1

### PR DESCRIPTION
**Pull request by lcarva**
This is the version we actually use in production.

### CODE REVIEW

**Review by yashvardhannanavati — APPROVED** [review]

File: docker/Dockerfile-workers [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN dnf -y install \
     && dnf clean all
 
 ENV OPM_SRC="/src"
-ENV OPM_TAG="v1.14.2"
+ENV OPM_TAG="v1.16.0"
```
Comment (yashvardhannanavati) [inline docker/Dockerfile-workers:29]: The one deployed in IIB is `v1.16.1`. Could you please update this?

**Comment by yashvardhannanavati** [comment]
+1 to merge after the change is made

File: docker/Dockerfile-workers (hunk 1 above)
Comment (lcarva) [reply docker/Dockerfile-workers:29]: Are you sure? On the ansible playbooks repo:
```
$ files/opm version
Version: version.Version{OpmVersion:"v1.16.0", GitCommit:"eb9fff5", BuildDate:"2021-02-09T13:53:02Z", GoOs:"linux", GoArch:"amd64"}
```

Comment (yashvardhannanavati) [reply docker/Dockerfile-workers:29]: That is strange. I'm pretty sure I updated it to 1.16.1: https://gitlab.cee.redhat.com/ansible-playbooks/iib-playbooks/-/merge_requests/126

**Comment by lcarva** [comment]
> +1 to merge after the change is made

Thanks @yashvardhannanavati. Merging now.

**Pull request merged**